test/IntegrationTests/MSTest.Acceptance.IntegrationTests/ThreadingTests.cs

```diff
@@ -225,7 +225,7 @@ public sealed class TestAssetFixture() : TestAssetFixtureBase(AcceptanceFixture.
         public const string STAThreadProjectName = "STATestThreading";
         public const string LifecycleAttributesVoidProjectName = "LifecycleAttributesVoid";
         public const string LifecycleAttributesTaskProjectName = "LifecycleAttributesTask";
-        public const string LifecycleWithParallelAttributesTaskProjectName = "LifecycleAttributesTask";
+        public const string LifecycleWithParallelAttributesTaskProjectName = "LifecycleWithParallelAttributesTask";
```

Comment:
This name was the same as above so we ended up writing into the same folder and mixing sources, added explicit check for that so it does not happen, and each asset gets its own folder that did not exist before. The names need to be unique only within the same asset group (produced by GetAssetsToGenerate method), not within the whole solution.